### PR TITLE
CONSOLE-4226: (deps) prep for webpack 5

### PR DESCRIPTION
**Pull request by logonoff**
<h2 id="feature-fix">CONSOLE Features and Fixes:</h2>

https://issues.redhat.com/browse/CONSOLE-4226

<h2 id="solution-description">Solution description</h2> 

- Update webpack related deps to the latest v4-supporting versions
- Update CopyWebpackPlugin to newest config syntax
- Replace HtmlWebpackExcludeAssetsPlugin with HtmlWebpackSkipAssetsPlugin because the former is no longer maintained / no longer works with newer versions of HtmlWebpackPlugin

### CODE REVIEW

**Comment by logonoff** [comment]
/hold

PR is still WIP and I want some input from the console team for potential implications/impact on related repos

**Comment by logonoff** [comment]
I also need to fix monaco because it's broken, however it builds now :)

image

File: frontend/webpack.config.ts [hunk 1]
```diff
@@ -304,42 +304,57 @@ const config: Configuration = {
         /vendor-patternfly-4-shared~main.*\.css/,
       ],
     }),
-    new HtmlWebpackExcludeAssetsPlugin(),
```
Comment (logonoff) [inline frontend/webpack.config.ts:307]: Removed as it no longer works with newer versions of the `HtmlWebpackPlugin`: see https://github.com/jamesjieye/html-webpack-exclude-assets-plugin/issues/18

**Review by vikram-raj — COMMENTED** [review]
/lgtm

**Comment by sanketpathak** [comment]
Cluster builds fine with the pr
/label qe-approved

**Pull request merged**